evm/src/Transceiver/WormholeTransceiver/WormholeTransceiver.sol

```diff
@@ -271,4 +273,32 @@ contract WormholeTransceiver is
         checkFork(wormholeTransceiver_evmChainId);
         return getWormholePeer(vm.emitterChainId) == vm.emitterAddress;
     }
+
+    function _encodePayload(
+        address caller,
+        bytes32 recipientNttManagerAddress,
+        bytes memory nttManagerMessage,
+        bool isSpecialRelayer
+    ) internal pure returns (TransceiverStructs.TransceiverMessage memory, bytes memory) {
+        // Transceiver payload is prefixed with 2 bytes
+        // representing the version of the payload.
+        // The rest of the bytes are the -actual- payload data.
+        // In payload v1, the payload data is a boolean
+        // representing whether the message should be picked
+        // up by the special relayer or not.
+        bytes memory transceiverPayload = abi.encodePacked(uint16(1), isSpecialRelayer);
```

Comment:
Why is the version a uint16? I assume a uint8 should be sufficient?

Reply:
you're right, there's no need for it to be uint16, uint8 is enough, I'm changing this